### ISSUE TITLE
Toots float at top of timeline

### ISSUE THREAD

**Issue by stefanhayden** [tootsuite/mastodon#5377]
Randomly tweets will float at the top of my federated timeline. The "relative time" is always "now". This example happened on chrome 61.0.3163.100 / windows 10. I do not know how to reproduce atm.

![example](https://user-images.githubusercontent.com/87616/31555075-a2ba6e2e-b00d-11e7-996e-2270b7fda3cf.gif)

* * * *

- [ ] I searched or browsed the repo’s other issues to ensure this is not a duplicate.
- [ ] This bug happens on a [tagged release](https://github.com/tootsuite/mastodon/releases) and not on `master` (If you're a user, don't worry about this).

**Comment by unarist** [comment]
Yeah, I also experienced this issue recently. This might be related to toot dupe bug on the timeline, which also happens sometimes on v2.0.0rc.

**Comment by unarist** [comment]
I got some information about my issue, and I think it's related to your one.

![image](https://user-images.githubusercontent.com/705555/31585826-4b6cb828-b203-11e7-9ea0-179f3aa8b9b5.png)

In this case, one status disappears, and same status appears many time below of it.

Statuses in the timeline should have same `aria-setsize` value and it will be updated when timeline being longer. However, bugged statuses have different values for it, and it won't be updated. Looks like those statuses are no longer controlled by React.

So floated statuses in your timeline were in same situation I think.

This might be related to React 16 which we've began to use from v2.0.0, but still I don't know the cause and reproducing steps...
